pkgs/build-support/rust/default-crate-overrides.nix

```diff
@@ -1,6 +1,7 @@
 { stdenv, pkgconfig, curl, darwin, libiconv, libgit2, libssh2,
```

Comment:
```suggestion
{ stdenv, pkg-config, curl, darwin, libiconv, libgit2, libssh2,
```

Reply:
This affects the whole file, so I put it into a second commit.